ziti-tunnel/cmd/ziti-tunnel/main.go

```diff
@@ -29,5 +36,59 @@ func init() {
 
 func main() {
 	debugz.AddStackDumpHandler()
+	logReleaseVersionCheck()
 	subcmd.Execute()
 }
+
+func logReleaseVersionCheck() {
```

Comment:
you could make this function take the constant that represents the name of the binary....

or use os.Args[0] and get the name of the executable and then it'd be fully dynamic.. something like:

import "path/filepath" and then use:
```
	exeName := filepath.Base(os.Args[0])
```

then you can put this whole function into the util pacakge and it'd be trivial to use from other CLI commands, right?

Reply:
if you go that route though, make sure you test what the behavior is when you rename the exe to something like `ztun` ... i expect this check stops working then. 

also any chance you could show the various outputs of these as screen caps? I'm somewhat too lazy to pull/build test the permutations but i'm interested to see what each look like

Reply:
![warn: current build is unreleased development build](https://user-images.githubusercontent.com/1434400/142926668-cbbe4a5c-3795-4d51-b37c-f12eda14c62d.png)

![update available: current build is superseded by latest](https://user-images.githubusercontent.com/1434400/142927534-399062e2-2771-4379-84a3-e44f7b92afb1.png)

![info: current build is latest](https://user-images.githubusercontent.com/1434400/142927685-46c64cdc-5870-4b1c-9f84-cc922519386e.png)

Reply:
If changed the log level for these messages from INFO and WARN to DEBUG so they will not normally be printed unless `--verbose`.

Reply:
![preview](https://user-images.githubusercontent.com/1434400/142936303-ac3eecd8-ddb6-41cb-b971-37fed50d94e6.png)